app/controllers/events/view/sessions/list.js

```diff
@@ -72,6 +77,36 @@ export default Controller.extend({
     viewSession(id) {
       this.transitionToRoute('events.view.sessions.edit', id);
     },
+    lockSession(session) {
+      session.set('isLocked', true);
+      this.set('isLoading', true);
```

Comment:
Everything looks good, but just had a question, wouldn't it be nice if the `session.set('isLocked', true)` be executed when isLoading is true? (I think it's just code design, won't affect the functionality, of course)

Reply:
@uds5501 I did this intentionally. I had a number of references where values were set before setting `isLoading` to true. As we know that `isLoading` is basically for loading status if a task time so it is added just before the time-taking task. However, if you want you can refer to `login-form` or `register-form` where similar implementations were carried out.

Reply:
Understood!